src/index.js

```diff
@@ -1 +1,97 @@
 import './style.css';
+
+// Id's
+const nameInpt = document.getElementById('nameInpt');
+const scoreInpt = document.getElementById('scoreInpt');
+const listContainer = document.getElementById('listContainer');
+const form = document.getElementById('form');
+const refreshBtn = document.getElementById('refreshBtn');
+
+// Leaderboard Class setting
+class Leaderboard {
+  constructor() {
+    this.URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+  }
+
+  // Start the API method
+  start = async (game) => {
+    const response = await fetch(this.URL, {
+      method: 'POST',
+      body: JSON.stringify({
+        name: game,
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    });
+    const data = await response.json();
+    return data;
+  }
+
+  // Get the Users and Scores from the API method
+  getUserAndScores = async (apiId) => {
+    const response = await fetch(`${this.URL}${apiId}/scores/`);
+    const data = await response.json();
+    return data;
+  }
```

Comment:
- [ ] (optional) It would be nice if the user did not have to refresh the screen to get the score. Think about implementing this